lambda-buffers-compiler/test/Test/Samples/Proto/Utils.hs

```diff
@@ -1,4 +1,4 @@
-module Test.Samples.Proto.Helpers (
+module Test.Samples.Proto.Utils (
```

Comment:
I'd like to have everything under test/Test/Utils folder. No need to say 'Samples/Proto' imo